### PR TITLE
Delete Meetup

### PR DESCRIPTION
**Pull request by exploreshaifali**

### CODE REVIEW

File: systers_portal/meetup/urls.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
         name='about_meetup_location'),
     url(r'^(?P<slug>\w+)/members/$', MeetupLocationMembersView.as_view(),
         name='members_meetup_location'),
+    url(r'^(?P<slug>\w+)/(?P<meetup_slug>[\w-]+)/delete/$', DeleteMeetupView.as_view(),
+        name='delete_meetup'),
     url(r'locations/$', MeetupLocationList.as_view(), name='list_meetup_location'),
-    url(r'^(?P<slug>[\w-]+)/(?P<meetup_slug>[\w-]+)/$', MeetupView.as_view(), name="view_meetup"),
+    url(r'^(?P<slug>\w+)/(?P<meetup_slug>[\w-]+)/$', MeetupView.as_view(), name="view_meetup"),
```
Comment (ana-balica) [inline systers_portal/meetup/urls.py]: So we don't match the `-` in the meetup location slug?

File: systers_portal/meetup/views.py [hunk 2]
```diff
@@ -1,7 +1,10 @@
+from django.core.urlresolvers import reverse
 from django.shortcuts import get_object_or_404
 from django.views.generic import TemplateView
+from django.views.generic import DeleteView
```
Comment (ana-balica) [inline systers_portal/meetup/views.py]: `from django.views.generic import TemplateView, DeleteView`

File: systers_portal/templates/meetup/meetup_confirm_delete.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{% extends "meetup/base.html" %}
```
Comment (ana-balica) [inline systers_portal/templates/meetup/meetup_confirm_delete.html]: Whether remove the meetup bar or change the level of the heading from h1 to h2.

File: systers_portal/meetup/views.py (hunk 2 above)
Comment (exploreshaifali) [inline systers_portal/meetup/views.py]: Done.

File: systers_portal/templates/meetup/meetup_confirm_delete.html (hunk 3 above)
Comment (exploreshaifali) [inline systers_portal/templates/meetup/meetup_confirm_delete.html]: Done. I should have also thought this :(

File: systers_portal/meetup/urls.py (hunk 1 above)
Comment (exploreshaifali) [inline systers_portal/meetup/urls.py]: Lets keep it, I am adding `-` for each meetup_location slug.

**Comment by ana-balica** [comment]
Merged :)

**Pull request closed without merging**